src/Pages/PlaylistPage/Components/PlaylistModal/PlaylistModal.jsx

```diff
@@ -0,0 +1,159 @@
+import { useState, useEffect } from "react";
+import {
+  usePlaylist,
+  useAuth,
+  useAlert,
+  usePlaylistModal,
+} from "../../../../CustomHooks";
+import { Modal } from "../../../../Components/UI";
+import "./PlaylistModal.css";
+
+export const PlaylistModal = ({ video }) => {
+  const [playlistInput, setPlaylistInput] = useState({
+    title: "",
+    description: "",
+    error: null,
+  });
+
+  const {
+    playlists,
+    getAllPlaylists,
+    createPlaylist,
+    addVideoToPlaylist,
+    deleteVideoFromPlaylist,
+  } = usePlaylist();
+
+  const {
+    authState: { token, isAuthenticated },
+  } = useAuth();
+
+  const { setShowAlert } = useAlert();
+  useEffect(() => {
+    console.log("effect playlist");
+    if (playlists === null) {
```

Comment:
Instead of checking if playlist is equal to null, you can check 
 if(!playlists)